frontend/messages/en.json

```
@@ -963,20 +963,152 @@
+	"privacy_legal_records": "Legal Records",
+	"privacy_criminal_records": "Criminal Records",
+	"privacy_judicial_data": "Judicial Data",
+	"privacy_preferences": "Preferences",
+	"privacy_opinions": "Opinions",
+	"privacy_feedback": "Feedback",
+	"privacy_images_photos": "Images and Photos",
+	"privacy_voice_recordings": "Voice Recordings",
+	"privacy_video_recordings": "Video Recordings",
+	"privacy_customer": "Customer/Client",
+	"privacy_prospect": "Prospective Customer/Client",
+	"privacy_employee": "Employee",
```

Biome (1.9.4)

[error] The key privacy_employee was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)